### PR TITLE
DEV: add RI debugging information

### PR DESCRIPTION
**Pull request by dwdougherty**

### CODE REVIEW

File: content/develop/connect/insight/debugging.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+weight: 2
+---
+
+If you are experiencing errors or other issues when using Redis Insight, follow the steps below to learn more about the errors and to identify root cause.
+
+## Connection issues
+
+If you experience connection issues, try these steps.
+
+### I. Launch Redis Insight in debug mode
```
Comment (andy-stark-redis) [inline content/develop/connect/insight/debugging.md:25]: Are the Roman numerals intentional?
```suggestion
### 1. Launch Redis Insight in debug mode
```

File: content/develop/connect/insight/debugging.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+* **macOS** (from the Applications folder):
+
+    `DEBUG=ioredis* open "Redis Insight.app"`
+
+* **Linux**:
+
+    `DEBUG=ioredis* "redis insight"`
+
+### II. Investigate logs
```
Comment (andy-stark-redis) [inline content/develop/connect/insight/debugging.md:41]:
```suggestion
### 2. Investigate logs
```

File: content/develop/connect/insight/debugging.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+You can review the Redis Insight log files (files with a `.log` extension) to get detailed information about system issues.
+These are the locations on supported platforms:
+
+- **Docker**: In the `/data/logs` directory *inside the container*.
+- **macOS**: In the `/Users/<your-username>/.redis-insight` directory.
+- **Windows**: In the `C:\Users\<your-username>\.redis-insight` directory.
+- **Linux**: In the `/home/<your-username>/.redis-insight` directory.
+
+## Other issues
+### To debug issues other than connectivity:
```
Comment (andy-stark-redis) [inline content/develop/connect/insight/debugging.md:52]: This one has a colon while the others don't (or of course, you could add a colon to the others if that works better).
```suggestion
### To debug issues other than connectivity
```

**Review by andy-stark-redis — APPROVED** [review]
Mostly fine, but a few things you might want to look at.

**Review by ViktarStarastsenka — APPROVED** [review]
LGTM

**Pull request merged**